README.md

```diff
@@ -30,7 +34,9 @@
  * Click on your new 'Settings' tab, and rename your new repo
  
 #### Clone the Code
-Run `git clone --recursive https://github.com/discordpp/echo-bot.git` in your desired directory. If you made a fork, replace `discordpp` and `echo-bot` as applicable.
+Run `git clone --recursive https://github.com/discordpp/echo-bot.git` in your desired directory.
+Make sure to run `git submodule update --init --recursive` in the new echo-bot directory to make sure it downloads all submodules.
```

Comment:
`git submodule update --init --recursive` is not required if you `clone` with `--recursive`